packages/bundler/src/modules/BundleManagerRIP7560.ts

```diff
@@ -94,4 +125,15 @@ export class BundleManagerRIP7560 extends BundleManager implements IBundleManage
   async getPaymasterBalance (paymaster: string): Promise<BigNumber> {
     return await this.provider.getBalance(paymaster)
   }
+
+  private computeBundleHash (userOps: OperationBase[]): string {
```

Comment:
this is a hash for a new entity that doesn't go on-chain. is it not needed

Reply:
Why not? What should we use as a bundle identifier instead? I think we need a way to identify a bundler, as I currently use `eth_getRip7560BundleStatus` to check if it was included, I think.